src/Screens/Main/Settings/Password.tsx

```diff
@@ -0,0 +1,232 @@
+          {pipe(
+            requestState,
+            RD.fold(
+              () => (
+                <>
+                  <NamedInputField
+                    style={styles.field}
+                    name="main.settings.account.password.current"
+                    isPasswordInput={true}
+                    value={currentPassword}
+                    onChangeText={setCurrentPassword}
+                    testID="main.settings.account.password.current"
+                  />
+                  <NamedInputField
+                    style={styles.field}
+                    name="main.settings.account.password.new"
+                    isPasswordInput={true}
+                    value={newPassword}
+                    onChangeText={setNewPassword}
+                    testID="main.settings.account.password.new"
+                  />
+                  <NamedInputField
+                    style={styles.field}
+                    name="main.settings.account.password.repeat"
+                    isPasswordInput={true}
+                    value={repeatedNewPassword}
+                    onChangeText={setRepeatedNewPassword}
+                    testID="main.settings.account.password.repeat"
+                  />
+                  <RN.View style={styles.buttonContainer}>
+                    <Button
+                      style={styles.cancelButton}
+                      messageStyle={styles.cancelButtonText}
+                      onPress={onGoBack}
+                      messageId="meta.cancel"
+                      testID="main.settings.account.password.cancel"
+                    />
+                    <Button
+                      style={styles.changePasswordButton}
+                      messageStyle={styles.buttonText}
+                      onPress={onButtonPress}
+                      messageId="meta.save"
+                      disabled={!isOkay}
+                      testID="main.settings.account.password.save"
+                    />
+                  </RN.View>
+                </>
+              ),
+              () => <Spinner style={styles.spinner} />,
+              () => (
+                <AlertBox
+                  imageStyle={styles.failBox}
+                  imageSource={require('../../images/alert-circle.svg')}
+                  duration={state.coolDownDuration}
+                  messageId="main.settings.account.password.failure"
+                />
+              ),
+              () => (
+                <AlertBox
+                  imageStyle={styles.successBox}
+                  imageSource={require('../../images/checkmark-circle-outline.svg')}
+                  duration={state.coolDownDuration}
+                  messageId="main.settings.account.password.success"
+                />
+              ),
+            ),
+          )}
+          <CreatedBySosBanner />
+        </SafeAreaView>
+      </RN.ScrollView>
+    </RN.View>
+  );
+};
+
+const styles = RN.StyleSheet.create({
```

Comment:
It's good to favor multiplies of 8 if possible. Works more consistenlty on different DPIs